graphGen.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/bash
+
+USE_FPS=0
+
+mkdir -p plot-data
+cd plot-data
+
+while getopts ":f" opt
+do
+    case $opt in
+        f)
+            USE_FPS=1
+            break
+            ;;
+        \?)
+            echo "Invalid option -$OPTARG" >&2
+            exit 1
+            ;;
+    esac
+done
+
+shift $((OPTIND-1))
+
+for log in "$@"
+do
+    tracker_version=${log#*-}
+    tracker_version=${tracker_version%/*}
+
+    dataset=${log#*/}
+    dataset=${dataset#*-}
+    dataset=${dataset%-*}
+
+    mkdir -p "$dataset"
+
+    cd "$dataset"
+
+    arguments=${log%.*}
+    arguments=${arguments#*/*-*-}
+
+    mkdir -p "$arguments"
+
+    cd "$arguments"
+
+    data_file="$tracker_version".dat
+    touch "$data_file"
+
+    echo "${tracker_version^}" >> "$data_file"
+
+    while IFS= read -r line 
+    do
+        #Skip last line
+        if [[ $line != *"->"* ]]; then
+           continue
+        fi
+        time=${line%,*,*}
+        time=${time//[!.0-9]/}
+        if [ "$USE_FPS" -eq "1" ]; then
+            time=$(bc <<< "scale=3;(1/$time)*1000;")
```

Comment:
You can do this easily in gnuplot. Instead of `plot file using using (i):i`, you can do `plot file using (i):(1000/column(i))`